examples/openspending/pages/index.tsx

```diff
@@ -32,15 +32,20 @@ export async function getStaticProps() {
     //  TODO: title should be the full name
     .map((code) => ({ code, title: code }));
 
+  const minPeriod = projects.map(project => project.fiscalPeriod ? project.fiscalPeriod.start : null).filter(item => item !== null).sort()[0]
```

Comment:
There's one item on the issue about fixing the GIFT datasets. Are we going to do that or should we move this item to a new issue?

Reply:
Noted. We can keep an inbox with the pending items and remove those from the issue.